content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,490 @@
+---
+title: "COVID-19 RDT Reference Application"
+linkTitle: "COVID-19 RDT"
+weight: 
+description: >
+ A customizable CHT application with support for 3rd party RDT integration   
+relatedContent: >
+---
+
+Medic has worked with [FIND](https://www.finddx.org) to provide a reference application of rapid diagnostic tests (RDTs) for COVID-19 in the CHT.  By leveraging the expertise of third-party applications, like [Dimagi's RD-Toolkit](https://github.com/dimagi/rd-toolkit/), you can customize this application in your current or future deployment of the CHT.
```

Comment:
Minor adjustment to language, and I think it's clearer to define the acronym without the `s` -- not sure if there is a standard way for that which we should follow though.

```suggestion
Medic has worked with [FIND](https://www.finddx.org) to provide a reference application using rapid diagnostic tests (RDT) for COVID-19 testing with the CHT.  By leveraging the expertise of third-party applications, like [Dimagi's RD-Toolkit](https://github.com/dimagi/rd-toolkit/), you can customize this application in your current or future deployment of the CHT.
```

Reply:
Yes! I went back and forth with the `s`, but happy to defer to your preference

Reply:
huh - couldn't accept your suggestion - exact same suggestion applied in 5d3f256